src/components/Chat.js

```diff
@@ -189,31 +193,67 @@ class TextComposer extends React.Component {
     const { altKey, shiftKey } = event;
     return (this._enterPressed(event) && (altKey || shiftKey));
   }
-
-
+ 
+  _handleSendButton () {
+    this._send();
+  }
+ 
   render() {
-    const { children } = this.props;
+    const { children, iconFSData } = this.props;
+    const iconContext = {
+       viewBox : iconFSData ? iconFSData.viewBox : '0 0  0 0',
+       width : iconFSData ? iconFSData.width : 0,
+       height : iconFSData ? iconFSData.height : 0,
+       pathD : iconFSData ? iconFSData.pathD : '',
+       sendIndicator : iconFSData ? '' : 'Send'
+    };
+
     const context = {
       value: this.state.value,
       onButtonClick: this._handleButtonClick,
 			onChange: this._handleOnChange,
 			onKeyDown: this._handleKeyDown
     };
+    const style = {
+        opacity: !context.value ? "0" : "1",
+        visibility: !context.value ? "hidden" : "visible"
+   };
 
     return (
       <TextComposer.Context.Provider value={context}>
         <StyledTextComposer {...this.props}>
           {children}
+          <StyledSendButton style={style} disabled={!context.value} onClick={this._handleSendButton} >{iconContext.sendIndicator}
+              <svg aria-hidden="true" role="img" xmlns="http://www.w3.org/2000/svg" viewBox={iconContext.viewBox} width={iconContext.width} height={iconContext.height}>
+              <path fill="currentColor" d={iconContext.pathD}/>
+              </svg>
+          </StyledSendButton>
         </StyledTextComposer>
       </TextComposer.Context.Provider>
     );
   }
 }
 
+const StyledSendButton = styled.button`
+    align-self: flex-end;
+    font-size: 1em;
+    width: auto;
+    border: none;
+    background-color: transparent;
+    outline: 0;
+    opacity: 0;
+    transition: 0.3s;
+    cursor: pointer; 
+
+  ${props => {
```

Comment:
Why do you need an entire code block here?  and calling `Object.assign` with only one parameter?
All you want to do here is return the the theme's send button css:
`${props => props.theme.Sendbutton.css; }`
like in most other places in code...